main.rb

```diff
@@ -163,13 +163,19 @@ def appstore_version
   bundle_id = env_has_key('AC_BUNDLE_ID')
   country = get_env('AC_APPSTORE_COUNTRY')
   uri = if country
-          URI("http://itunes.apple.com/lookup?bundleId=#{bundle_id}&country=#{country}")
+          puts "Fetching version from App Store for Bundle Identifier: #{bundle_id} and country: #{country}"
+          URI("https://itunes.apple.com/lookup?bundleId=#{bundle_id}&country=#{country}")
         else
-          URI("http://itunes.apple.com/lookup?bundleId=#{bundle_id}")
+          puts "Fetching version from App Store for Bundle Identifier: #{bundle_id}"
+          URI("https://itunes.apple.com/lookup?bundleId=#{bundle_id}")
         end
```

Comment:
@boztopuz avoid the repetition (DRY). Could you use the following logic?
```suggestion
puts "Fetching version from App Store for Bundle Identifier: #{bundle_id}" +
     (country ? " and country: #{country}" : "")

url = "https://itunes.apple.com/lookup?bundleId=#{bundle_id}"
url += "&country=#{country}" if country
uri = URI(url)
```